src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -487,3 +488,19 @@ export const selectSecondHopSellAccountId = createSelector(
 export const selectHopExecutionMetadata = createSelector(selectTradeQuoteSlice, swappers => {
   return [swappers.tradeExecution.firstHop, swappers.tradeExecution.secondHop]
 })
+
+export const selectHopStreamingSwapMetadata = createSelector(
+  selectHopExecutionMetadata,
+  (hopExecutionMetadata): StreamingSwapMetadata[] => {
+    const defaultValue = {
+      attemptedSwapCount: 0,
+      totalSwapCount: 0,
+      failedSwaps: [],
+    }
+
+    return [
+      hopExecutionMetadata[0].streamingSwap ?? defaultValue,
+      hopExecutionMetadata[1].streamingSwap ?? defaultValue,
+    ]
```

Comment:
`createDeepEqualOutputSelector` instead of `createSelector` to avoid wrongly invalidating the output reference?

Reply:
Also not sure if this is expected, but currently unused